designguidelines.md

```diff
@@ -0,0 +1,15 @@
+These components are used as the building blocks for creating all types of forms. These forms should be able to be used on all devices ie desktop, mobile and tablet.
+
+## Form rules:
+
+*Labels* - Always label the form component ie a text input, select box to help the user understand it’s meaning.
```

Comment:
It's not meant to be a list. Just a series of lines. Maybe I should just add a newline between each